services/src/main/resources/locale/portlet/news/News_en.properties

```diff
@@ -88,15 +88,15 @@ news.activity.clickToShowDetail=Click here for more news details
 
 news.archive.btn.confirm=Confirm
 news.archive.btn.cancel=Cancel
-news.archive.action=Archive article
+news.archive.action=Bookmark it to easily find it when searching
```

Comment:
Why this value is changed ?

Reply:
This was requested in the task, but if you wish, we can keep the old text.

Reply:
can you confirm that the key news.archive.action correspond to "Favorite" action ? 
It seems related to different things

Reply:
In addition, labels like that must be changed in crowdin

Reply:
As you said preferably that we add another key to avoid changing another label so it must that I add another key and I would change the description of the PR.